docker/hvd/Dockerfile.hvd-base

```diff
@@ -3,15 +3,18 @@
 
 ARG PTH_VERSION
 
-FROM pytorch/pytorch:${PTH_VERSION}-devel as builder
+FROM pytorch/pytorch:${PTH_VERSION}-devel AS builder
 
 ARG HVD_VERSION
 
 # Build Horovod
 RUN apt-get update && apt-get install -y git && \
-        git clone --recursive --depth 1 --branch ${HVD_VERSION} https://github.com/horovod/horovod.git /horovod && \
-        conda install -y cmake nccl -c conda-forge && \
+        conda install -y cmake=3.* nccl -c conda-forge && \
+        git clone --depth 1 https://github.com/horovod/horovod.git /horovod && \
         cd /horovod && \
+        git fetch --tags && \
+        git checkout ${HVD_VERSION} && \
+        git submodule update --init --recursive && \
```

Comment:
Can't we do that in one line as previously?

Reply:
The command was not working as expected I tried different variations locally and this is a bit longer, but it results in the correct branch being checked out.